src/main/java/com/mycompany/exercisetwo/Humming.java

```diff
@@ -0,0 +1,50 @@
+package com.mycompany.exercisetwo;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+/**
+ *
+ * @author usuario
+ */
+public class Humming extends Bird implements Death {
+
+    @Override
+    public void sleep() {
+        System.out.println("The humming usually sleep few hours");
+    }
+
+    @Override
+    public String die(LocalDate born, LocalDate die, String deathCause) {
+
+        Period period = Period.between(born, die);
+
+        return "The date of it's die is: " + die + ", lived: " + period.getYears() + " years, and the death     cause was: " + deathCause;
```

Comment:
What is that whole space between "and the death      cause was"?

Reply:
My bad, corrected.